pkg/cmd/server/start/start_master.go

```diff
@@ -234,7 +234,7 @@ func (o MasterOptions) RunMaster() error {
 			return err
 		}
 
-		fmt.Fprintf(o.Output.Get(), "Wrote master config to: %s\n", o.MasterArgs.GetConfigFileToWrite())
+		fmt.Fprintf(o.Output, "Wrote master config to: %s\n", o.MasterArgs.GetConfigFileToWrite())
```

Comment:
If that's true it means people haven't tested any of the code which I consider a developer problem, not a code problem.

> On Jun 5, 2015, at 7:42 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/cmd/server/start/start_master.go:
> 
> > @@ -234,7 +234,7 @@ func (o MasterOptions) RunMaster() error {
> >  			return err
> >  		}
> >  
> > -		fmt.Fprintf(o.Output.Get(), "Wrote master config to: %s\n", o.MasterArgs.GetConfigFileToWrite())
> > +		fmt.Fprintf(o.Output, "Wrote master config to: %s\n", o.MasterArgs.GetConfigFileToWrite())
> This will panic on nil Output which is a real pain.
> 
> —
> Reply to this email directly or view it on GitHub.
>